src/test/java/io/strimzi/kafka/bridge/mqtt/core/MqttKafkaMapperTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.core;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.strimzi.kafka.bridge.mqtt.utils.MappingRule;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Comparator;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * Unit tests for {@link MqttKafkaMapper}
+ */
+public class MqttKafkaMapperTest {
+
+    private ArrayList<MappingRule> mappingRules;
+    private MqttKafkaMapper mqttKafkaMapper;
+    //define the mapping rules as a JSON string. Add more rules if needed.
+    private static final String TOPIC_MAPPING_RULES_JSON =
+            "[" +
+                    "{\"kafkaTopic\":\"building_{building}_room_{room}\",\"mqttTopic\":\"building/{building}/room/{room}/#\"}," +
+                    "{\"kafkaTopic\":\"sensor_data\",\"mqttTopic\":\"sensors/+/data\"}," +
+                    "{\"kafkaTopic\":\"devices_{device}_data\",\"mqttTopic\":\"devices/{device}/data\"}," +
+                    "{\"kafkaTopic\":\"fleet_{vehicle}\",\"mqttTopic\":\"fleet/{fleet}/vehicle/{vehicle}/#\"}," +
+                    "{\"kafkaTopic\":\"building_{building}_others\",\"mqttTopic\":\"building/{building}/#\"}," +
+                    "{\"kafkaTopic\":\"sensor_others\",\"mqttTopic\":\"sensors/#\"}," +
+                    "{\"kafkaTopic\":\"building_others\",\"mqttTopic\":\"building/#\"}" + "]";
+
+    /**
+     * Load all the mapping rules before running the tests.
+     */
+    @Before
+    public void setUp() {
+        ObjectMapper mapper = new ObjectMapper();
+        try {
+            mappingRules = mapper.readValue(TOPIC_MAPPING_RULES_JSON, new TypeReference<>() {
+            });
```

Comment:
Any reason this isn't with the line above?

Reply:
Maybe something like this is more meaninful ...

```java
mappingRules = mapper.readerForListOf(MappingRule.class).readValue(TOPIC_MAPPING_RULES_JSON);
```

Reply:
@kyguy sorry, I did not get the question.

Reply:
If you are referring to line 46, I think the IDE's code reformat did it.

Reply:
@ppatierno magic.

Reply:
I think Kyle was asking why the `});` is not at the end of line 45.
I think this is just the IDE because you are declaring an anonymous class and IDE just moves the closed parenthesis to the next line. Anyway I would follow my suggestion instead of having this anonymous class.

Reply:
@ppatierno I think so, I have fixed that.